src/Microsoft.DotNet.Interactive.Formatting/DefaultPlainTextFormatterSet.cs

```diff
@@ -79,39 +97,53 @@ internal class DefaultPlainTextFormatterSet
                         var array = toArray.Invoke(null, new[] { obj });
 
                         writer.Write(array.ToDisplayString(PlainTextFormatter.MimeType));
+                        return true;
                     }),
 
-                new PlainTextFormatter<TextSpan>((span, writer) => writer.Write(span.ToString(OutputMode.Ansi))),
+
+                new PlainTextFormatter<TextSpan>((context, span, writer) =>
+                    {
+                        writer.Write(span.ToString(OutputMode.Ansi));
+                        return true;
+                    }),
 
                 // Newtonsoft.Json types -- these implement IEnumerable and their default output is not useful, so use their default ToString
-                new PlainTextFormatter<Newtonsoft.Json.Linq.JArray>((obj, writer) => writer.Write(obj)),
+                new PlainTextFormatter<Newtonsoft.Json.Linq.JArray>((context, obj, writer) =>
+                    {
+                        writer.Write(obj);
+                        return true;
+                    }),
 
-                new PlainTextFormatter<Newtonsoft.Json.Linq.JObject>((obj, writer) => writer.Write(obj)),
+                new PlainTextFormatter<Newtonsoft.Json.Linq.JObject>((context, obj, writer) =>
```

Comment:
`JObject` and `JArray` both derive from `JToken`, so this and the previous formatter can be combined.

Reply:
got it thanks, will be in next update